gtsfm/runner/plot_metrics.py

```diff
@@ -0,0 +1,47 @@
+"""Script to generate a report of metrics with tables and plots 
+using the metrics that have been logged as JSON in a previous run of the pipeline. 
+
+Authors: Akshay Krishnan
+"""
+import os
+import argparse
+
+from gtsfm.evaluation.metric import GtsfmMetric, GtsfmMetricsGroup
+import gtsfm.evaluation.metrics_visualizer as metrics_viz
+import gtsfm.utils.logger as logger_utils
+
+logger = logger_utils.get_logger()
+
+
+def create_metrics_plots_html(json_path: str, output_dir: str) -> None:
+    """Creates a HTML report of metrics from frontend, avergaging, data association and bundle adjustment.
+
+    Reads the metrics from JSON files in a previous run.
+
+    Args:
+        json_path: Path to folder that contains metrics json files.
+        output_dir: directory to save the report, uses json_path if empty.
+    """
+    metrics_groups = []
+    GTSFM_MODULE_METRICS_FNAMES = [
+        "frontend_summary.json",
+        "multiview_optimizer_metrics.json",
+        "data_association_metrics.json",
+        "bundle_adjustment_metrics.json",
+    ]
+    for filename in GTSFM_MODULE_METRICS_FNAMES:
+        logger.info("adding plot for %s", filename)
```

Comment:
we might not always have plots, right?

Reply:
why wouldnt we? it would be blanch if no visitors

Reply:
Oh yeah, you mean if all metrics are scalar?

Reply:
updating that comment